Makefile

```diff
@@ -169,6 +169,7 @@ dist/argo-linux-amd64: GOARGS = GOOS=linux GOARCH=amd64
 dist/argo-darwin-amd64: GOARGS = GOOS=darwin GOARCH=amd64
 dist/argo-windows-amd64: GOARGS = GOOS=windows GOARCH=amd64
 dist/argo-linux-arm64: GOARGS = GOOS=linux GOARCH=arm64
+dist/argo-darwin-arm64: GOARGS = GOOS=darwin GOARCH=arm64
```

Comment:
minor - sort

Reply:
Would you like me to sort all `dist/something` alphabetically?

Reply:
I've ordered them alphabetically now, but putting `linux` first, followed by `darwin` and `windows` (same as the line further down).